content/verification/veteran-verification/release-notes/2023-05-18

```diff
@@ -0,0 +1,26 @@
+We added a new version (v2) of the Veteran Verification API to the developer portal, and the API has been renamed to the 'Veteran Service History and Eligibility API'.
+
+The following enhancements and new features are available in v2:  
+
+All of the API's endpoints can now be accessed using either Authorization Code Flow OAuth or Client Credentials Grant (CCG) OAuth.   
```

Comment:
'Authorization Code Flow' and 'Client Credentials Grant' will be hyperlinked (as shown here: https://community.max.gov/pages/viewpage.action?pageId=2329056319)?